templates/search/v710/contentFilterPublished.tmpl

```diff
@@ -0,0 +1,36 @@
+{
+  "bool": {
+    "should":[
+          {
+            "bool": {
+              "must_not": {
+                "term": {
+                  "description.cancelled": true
```

Comment:
Is it worth updating this? Remove description object as model has been flattened

Reply:
fixed them.